tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -186,7 +201,7 @@ jobs:
         inputs:
           command: 'custom'
           workingDir: ${{ parameters.testWorkspace }}
-          customCommand: 'install $(Initialize.testPackageTgz) @ff-internal/aria-logger'
+          customCommand: 'install $(Initialize.testPackageTgz) ${{ parameters.loggerPackage }}'
```

Comment:
Sending DDS fuzz test telemetry to our preprod tenant for debugging is an interesting idea, but I'm not sure it would be particularly valuable in the current state of the world (fuzz tests are pretty deterministic, just debugging the test is a totally viable diagnostic strategy). The bigger problem here is actually that this pipeline as well as the real service pipeline / stress pipeline seem to always be getting @ff-internal/aria-logger@0.0.11 (a version from march 2021), and that version isn't installable in test branches due to our package retention policies. We should fix this--going to file a work item before completing this pr.

Reply:
I misremembered the scenario slightly. What I said about the pipeline not working in test branches is accurate, but that version of aria-logger is actually the most recent one. The problem is it depends on really old @fluidframework packages, which is what causes the issue in feeds with retention policies. (some of this has changed somewhat recently, see #11623 ).

In any case, I think not installing the logger on  this pipeline is fine.